prefs/HBLinkTableCell.m

```diff
@@ -28,6 +28,16 @@ - (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSStr
 		self.detailTextLabel.numberOfLines = _isBig ? 0 : 1;
 		self.detailTextLabel.text = specifier.properties[@"subtitle"] ?: @"";
 		self.detailTextLabel.textColor = IS_IOS_OR_NEWER(iOS_7_0) ? [UIColor systemGrayColor] : [UIColor tableCellValue1BlueColor];
+		if (@available(iOS 13.0, *)) {
+			if (IS_IOS_OR_NEWER(iOS_13_0)) {
+				self.tintColor = [UIColor labelColor];
+			}
+		}
+		if (@available(iOS 13.0, *)) {
+			if (IS_IOS_OR_NEWER(iOS_13_0)) {
+				self.detailTextLabel.textColor = [UIColor secondaryLabelColor];
```

Comment:
Why don't you combine this statement into the above one so that you do only one iOS version check?

Reply:
`@available()`, crazily, always returns true on iOS before about 8.0! Seems like its fallback should be returning false, but it does the opposite. So it needs to be doubly checked. One makes the compiler happy (since I’d rather be wrapping newer API usage in `@available()` anyway), the other is the runtime check that’s fully compatible before 8.0.

Reply:
I mean, move those two color setting statements to be together, not separating them.

Reply:
Oh, yep, it should be that way. I think I told kritanta that I’d clean it up when I merge.